preload.php

```diff
@@ -2,240 +2,240 @@
 
 declare(strict_types = 1);
 
-require_once __DIR__ . '/vendor/nikic/php-parser/lib/PhpParser/Lexer.php';
```

Comment:
We'll need to load own `preload.php` anyway to make sure it won't conflict with project's `php-parser`, make this CI to pass:
https://github.com/rectorphp/rector-prefixed/runs/2518375775

Not sure how to solve that now :/